internal/api/token_refresh.go

```diff
@@ -50,7 +51,7 @@ func (a *API) RefreshTokenGrant(ctx context.Context, w http.ResponseWriter, r *h
 	for retry && time.Since(retryStart).Seconds() < retryLoopDuration {
 		retry = false
 
-		user, _, session, err := models.FindUserWithRefreshToken(db, params.RefreshToken, false)
+		user, token, session, err := models.FindUserWithRefreshToken(db, params.RefreshToken, false)
```

Comment:
we wouldn't need to use the `token` here if we go with using the `session.CreatedAt` if the `RefreshedAt` is nil

Reply:
decided to keep this - https://github.com/supabase/gotrue/pull/1298#discussion_r1383865526